helpers/calendar/events.py

```diff
@@ -90,3 +90,50 @@ def get_all_room_schedules(self, query, start_date, end_date):
                 all_events.append(current_event)
 
         return all_events, all_dates
+
+    def get_room_index(self, attendees, calendar_id):
+        index_of_room = 0
```

Comment:
What is the significance of index_of_room? What does it represent?

Reply:
When we get the data from google API the room is part of a list of attendees. The index of room is the position of the room on the list of attendees. Wee need this since the way we cancel a room meeting on the google calendar is by updating a field on the room under attendees that makes the room decline attending the meeting so as to free it up for more bookings.

Reply:
> When we get the data from google API the room is part of a list of attendees. The index of room is the position of the room on the list of attendees. Wee need this since the way we cancel a room meeting on the google calendar is by updating a field on the room under attendees that makes the room decline attending the meeting so as to free it up for more bookings.

Then have a more descriptive function name like get_room_index_from_attendees

Reply:
I have implemented this @Bonifase